schema/schema.go

```diff
@@ -27,6 +27,21 @@ const (
 	MediaTypeManifestList Validator     = v1.MediaTypeImageManifestList
 	MediaTypeImageConfig  Validator     = v1.MediaTypeImageConfig
 	MediaTypeImageLayer   unimplemented = v1.MediaTypeImageLayer
+
+	// MediaTypeDockerImageManifestList specifies the docker image's mediaType for manifest lists.
+	MediaTypeDockerImageManifestList = "application/vnd.docker.distribution.manifest.list.v2+json"
+
+	// MediaTypeDockerImageManifest specifies the docker image's mediaType for the current version.
+	MediaTypeDockerImageManifest = "application/vnd.docker.distribution.manifest.v2+json"
+
+	// MediaTypeDockerImageLayer specifies the docker image's mediaType used for layers referenced by the manifest.
+	MediaTypeDockerImageLayer = "application/vnd.docker.image.rootfs.diff.tar.gzip"
+
+	// MediaTypeDockerImageConfig specifies the docker image's mediaType for the image configuration.
+	MediaTypeDockerImageConfig = "application/vnd.docker.container.image.v1+json"
+
+	// MediaTypeNonDistributable specifies the non-distributable layer's mediaType.
+	MediaTypeNonDistributable = "application/vnd.oci.image.layer.nondistributable.v1.tar+gzip"
```

Comment:
I still don't think we want to export these as part of our public API (discussion in #320).